### PR TITLE
Capstone1: Artfifcial Intelligence and Iot forum

### PR DESCRIPTION
**Pull request by GedeonTS**
#  Artificial Intelligence and IoT forum

In this project, I created a responsive and dynamic webpage using Html, css, and javascript.

your review is welcome!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# Artficial-intelligence-forum
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Artficial-intelligence-forum
+
+> This is a Webpage about the international artificial intelligence forum.
+
```
Comment (sja-thedude) [inline README.md]: - [x] Kindly make sure to add an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+Click [here](https://gedeonts.github.io/Artificial-intelligence-forum/)
```
Comment (sja-thedude) [inline README.md]: - [x] Kindly fix this. This link is broken.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+            <a href="#portfolio" class="menu-list" id="elem1">UTI Projects</a>
+          </li>
+          <li>
+            <a href="./about-page.html" class="menu-list" id="elem2">About</a>
+          </li>
+          <li>
+            <a href="#contact-me" class="menu-list nav-styled" id="elem3">Partners</a>
+          </li>
+        </ul>
+      </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly add the top dark navbar which contains the social media icons and the logout button aligned to the right as per the original design. Also make sure to place this in `nav` tags.

Screenshot 2022-04-15 at 2 42 59 PM

- [ ] Kindly make sure that the background colour of the hamburger menu is the same as the colours used on the home page. 

Screenshot 2022-04-15 at 2 41 17 PM

File: index.html [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+        </div>
+        <footer class="footer-wrapper">
+          <div class="footer">
+            <div class="footer-logo">
+              <img class="logo-footer" src="./IMG/itu-logo.png" alt="logo" />
+            </div>
+            <div class="footer-content">
+              <p>© 2022 The ITU-T. All rights reserved.</p>
+              <p class="footer-extra">全球人工智能和物聯網會議是一個領先的論壇，供來自學術界</p>
+            </div>
```
Comment (sja-thedude) [inline index.html:189]: - [ ] Kindly make sure to align the logo to the left as per the original design.
Screenshot 2022-04-15 at 2 58 10 PM

File: about-page.html [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+        </div>
+        <div class="london">
+          <div class="inside">
+            <h2 class="date">2017</h2>
+            <p>AI and Iot Forum in London</p>
+          </div>
+        </div>
+      </div>
+      <div class="partners partnersa">
+        <h2 class="logo-header font">Partners</h2>
```
Comment (sja-thedude) [inline about-page.html]: - Kindly uses semantic tags, instead of divs use more of `section` for best practices.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,825 @@
...
+
+#speaker-list2 {
+  display: none;
+}
+
+.footer-extra {
+  display: none;
+}
+
+/* media media media media media media media media media media media media media media media media media media  */
```
Comment (sja-thedude) [inline style.css]: - [ ] Kindly remove the commented out code for the clean codebase.

File: script.js [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+  Name: 'Ms Mame Aminata Diambo',
+  Work: 'Student',
+  Descrition: 'Student, ESMT Dakar, Senegal',
+}, {
+  photo: './IMG/speakers/allo-cin2.jpg',
+  Name: 'Mr Lacina Koné',
+  Work: 'CEO of Smart Africa',
+  Descrition: ' Advisor to the Prime Minister of the Republic of Côte d`Ivoire (2017-2018), in charge of Digital Transformation',
+}];
+
```
Comment (sja-thedude) [inline script.js:69]: - [x] Kindly make sure to add the corner checks background image to all the speakers. The background image is shown in the video you recorded but it is not visible on the Github page you deployed nor when I clone it on my mac.
- [ ] Kindly also follow the spacing and give the appropriate margin as per the original design.
- Original design:
Screenshot 2022-04-15 at 3 21 32 PM

- Your design:
Screenshot 2022-04-15 at 3 21 11 PM

File: index.html [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+          </div>
+          <div class="step ">
+            <span class="icon">
+              <i class="material-icons">assignment_ind</i>
+            </span>
+            <h4>
+              Info Session
+            </h4>
+            <div class="content">
+              Information on ITU Activities on "Environment, Climate Change and
```
Comment (sja-thedude) [inline index.html:140]: - [ ] Kindly make sure to follow the background color as per the original design.
- Original design:
Screenshot 2022-04-15 at 3 29 21 PM

- Your design:
Screenshot 2022-04-15 at 3 29 01 PM

File: about-page.html [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+            Internet of Things and Smart Cities and Communities (JCA-IoT and
+            SC&C) meeting which took place on 6 December 2018 in the same venue.
+          </p>
+        </section>
+        <section class="text2 contact-text-resize text2b">
+          <p>
+            Please contact us per Email for any further question about the ItU
+            Event, global forum on Artificial Intelligence and Internet of
+            things<br /><b><u>ituGlobalForum@itu.ca</u></b>
+          </p>
```
Comment (sja-thedude) [inline about-page.html]: - [ ] Kindly make sure to follow the background colour as given in the original design and also follow the spacing, font size for the webpage.

- Original design:
Screenshot 2022-04-15 at 3 33 33 PM

- Your design:
Screenshot 2022-04-15 at 3 33 53 PM

File: index.html [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+          <h3 class="greetings">"Hello!smart world"</h3>
+          <h1 class="top-header">GLOBAL FORUM ON AI AND IoT</h1>
+        </section>
+        <section class="text-container">
+          <p class="text1">
+            A platform to discuss trends in Artificial Intelligence, Internet of
+            Things and Smart Cities and discuss the role of artificial
+            intelligence and IoT to shape smarter and more sustainable cities
+            and to address urban challenges, with the final goal of improving
+            the quality of life of people.
```
Comment (sja-thedude) [inline index.html:81]: - [ ] Kindly make sure to follow the colour as per the original design for the heading. Use #ec5242.
- [ ] Kindly make sure that the hamburger is of #272a31 colour as per the original design.
- Original design:
Screenshot 2022-04-15 at 3 40 01 PM

- Your design:
Screenshot 2022-04-15 at 3 39 09 PM

File: README.md [hunk 11]
```diff
@@ -1 +1,52 @@
-# Artficial-intelligence-forum
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Artficial-intelligence-forum
+
+> This is a Webpage about the international artificial intelligence forum.
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly add more description of the project like "This is part of microverse curriculum and built for the capstone. This is a simple website that navigates from the home page to the about page. This is responsive and follow mobile first."

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @GedeonTS, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
- [ ] Good Video presentation. 👍 But I find some images were not displayed when I tried your website by cloning or with your Github page deploy. I see you gave the presentation via firefox. Kindly make sure that the website works fine on chrome as its default browser for most and is used by everyone.
- [ ] _Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by alexander16108 — APPROVED** [review]
Hi, @GedeonTS, 

good job implementing all the requested activities done in the project, 🥳 🍾  congratulations you can move on with the module ✔️ 

In my opinion, your project is ready There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: README.md (hunk 2 above)
Comment (GedeonTS) [reply README.md]: Testing thread behaviour

**Pull request merged**